base/exports.jl

```diff
@@ -1076,6 +1076,7 @@ export
 # help and reflection
     apropos,
     current_module,
+    editcmd,
```

Comment:
In fact my preference would be to entirely disallow net increases in the list of Base exports.